aptos-move/framework/aptos-framework/doc/aggregator_factory.md

```diff
@@ -44,9 +44,9 @@ should be destroyed by the user.
 
 
 <pre><code><b>use</b> <a href="aggregator.md#0x1_aggregator">0x1::aggregator</a>;
-<b>use</b> <a href="../../aptos-stdlib/../move-stdlib/doc/error.md#0x1_error">0x1::error</a>;
+<b>use</b> <a href="..\../aptos-stdlib\../move-stdlib\doc\error.md#0x1_error">0x1::error</a>;
```

Comment:
By any chance, are you using a Windows machine? It seems that all `/`s are replaced with `\`s in your commit. This may not pass the CI test. The CI test runs on a linux machine, and would expect `/` instead of `\`. (@wrwg )

Aptos's `dev_setup` (https://github.com/aptos-labs/aptos-core/blob/main/scripts/dev_setup.sh) supports Linux and Mac. I am wondering if either Linux or Mac is available for you so that you can setup the dev env and regenerate the doc?

Reply:
pls check.